### ISSUE TITLE
Core Site Kit Functionality | Disconnect User from Site Kit

### ISSUE THREAD

**Issue by bjackson27** [google/site-kit-wp#252]
Set up Core Site Kit Functionality, Disconnect User from Site Kit.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* User clicks "Disconnect" (drop-down from user's Google account)
    * user will see a warning ("Disconnecting Site Kit by Google will remove your access to all services. After disconnecting, you will need to re-authorize to restore service.")
    * if they proceed with disconnecting, they should be redirected to the WordPress dashboard

## Implementation Brief

* n/a

## Changelog entry

* Core Site Kit Functionality | Disconnect User from Site Kit

**Comment by adamsilverstein** [comment]
In my testing, this did not pass the Acceptance Criteria, however I think they may be written incorrectly.

The AC says:

> if they proceed with disconnecting, they should be redirected to the first step of the setup process

In my testing I was taken to the WordPress Dashboard - which I think is the current, expected behavior. when I clicked the Site Kit Dashboard link I was "the first step of the setup process" and then able to continue the re-authentication process. 

Screencast: https://cl.ly/b3ecd366ca9d

Can we change the AC here @felixarntz?

I also ran the E2E tests from https://github.com/google/site-kit-wp/pull/542 and test passed locally:

![image](https://user-images.githubusercontent.com/2676022/65554334-7e0aeb80-dee6-11e9-900d-80c165f4eeec.png)

cc: @aaemnnosttv
